esutil/bulk_indexer.go

```diff
@@ -106,6 +106,10 @@ type BulkIndexerItem struct {
 	Index           string
 	Action          string
 	DocumentID      string
+	Routing         string
+	DocumentType    string
```

Comment:
Correct me if i'm wrong, but i'm pretty sure DocumentType will be removed in v8

Reply:
Oops! I'm sorry.
Fixed.